### ISSUE TITLE
PS4 and PS5 cameras support.

### ISSUE THREAD

**Issue by Hackinside** [AIRLegend/aitrack#131]
Would be nice to use PS4 or PS5 cameras. Drivers are available [here](https://github.com/Hackinside/Playstation-camera-drivers).

**Comment by DragControl** [comment]
Yes, It would be great to be able to use Playstation cameras because it has a very low response time.

**Comment by AIRLegend** [comment]
Sorry, I don't have any PS4/5 cameras. What's the gain over exposing them as regular webcams on Windows and using them as usual?

**Comment by Hackinside** [comment]
> Sorry, I don't have any PS4/5 cameras. What's the gain over exposing them as regular webcams on Windows and using them as usual?

If you get a PS4 camera I can provide you with an adapter if you need it for your project.
